File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/using_private_runme.m
using_private

f = FooBar();
f.x = 3;

if (f.blah(4) != 4)
    error, "blah(int)"
endif

if (f.defaulted() != -1)
    error, "defaulted()"
endif

if (f.defaulted(222) != 222)
    error, "defaulted(222)"
endif

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/using_private_runme.go
package main

import . "./using_private"

func main() {
	f := NewFooBar()
	f.SetX(3)

	if f.Blah(4) != 4 {
		panic("blah(int)")
	}

	if f.Defaulted() != -1 {
		panic("defaulted()")
	}

	if f.Defaulted(222) != 222 {
		panic("defaulted(222)")
	}
}
